src/utils/FontManager.js

```diff
@@ -22,6 +22,7 @@ limitations under the License.
  */
 
 function safariVersionCheck(ua) {
+    console.log("Browser is Safari - checking version for COLR support");
```

Comment:
I would suggest also moving the error line `console.error("Couldn't determine Safari version to check COLR font support, assuming no.", err);` to the same path as the `return false`, so we also log something about the result.